com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Fixed
 
 - Fixed endless dialog boxes when adding a NetworkBehaviour to a NetworkManager or vice-versa (#1947)
+- `FixedString` types can now be used in NetworkVariables and RPCs again without requiring a `ForceNetworkSerializeByMemcpy<>` wrapper (#1961)
```

Comment:
Doesn't this PR also add support for Vector2Int etc?

Reply:
Yes it does. Fixed.